common-lib/src/cmr/common/test/test_check_ext.clj

```diff
@@ -51,6 +52,23 @@
 
           (chuck/pass? reports#))))))
 
+(defmacro qc-and-report-exception-with-seed
+  [name seed final-reports tests bindings & body]
```

Comment:
I copied the above function into the one that I modified that does the same thing.  I think when a test fails, it re-runs it and captures the information from the failed test so that it can generate a report on it.  I just figured out how to add the seed value so that the generated values for the UMM record can be repeated.  I will put this into the doc string.